roles/ocp_on_libvirt/tasks/redfish_setup.yml

```
@@ -0,0 +1,34 @@
+- name: Set hardware vendor to "kvm"
+  set_fact:
+    vendor: kvm
+    bmc_user: "{{ vbmc_user }}"
+    bmc_password: "{{ vbmc_pass }}"
+  loop: "{{ groups['nodes'] }}"
+  delegate_to: "{{ item }}"
+
+- name: Install sushy-tools
+  ansible.builtin.include_role:
+    name: setup_sushy_tools
+  vars:
+    sushy_tools_port: "{{ redfish_port }}"
+    repo_root_path: "{{ ocp_on_libvirt_repo_root_path }}"
+    cert_country: "{{ ocp_on_libvirt_cert_country }}"
+    cert_state: "{{ ocp_on_libvirt_cert_state }}"
+    cert_locality: "{{ ocp_on_libvirt_cert_locality }}"
+    cert_organization: "{{ ocp_on_libvirt_cert_organization }}"
+    cert_organizational_unit: "{{ ocp_on_libvirt_cert_organizational_unit }}"
+    sushy_ignore_boot_device: false
```

I think you want to inject here the role's variable, from my comment above. It would be something like `ocp_on_libvirt_sushy_ignore_boot_device` or something shorter.

```suggestion
    sushy_ignore_boot_device: "{{ ocp_on_libvirt_sushy_ignore_boot_device | bool }}"
```

Otherwise the variable in this role is never used

---

Thanks for the catch.

The thing is, the sushy_ignore_boot_device must always be set to false for the solution to work.

The way I see it, it'd be simpler just to remove the variable from the defaults and the documentation.

I don't know why I added it to the defaults in the first place.